quic/s2n-quic-core/src/inet/ipv6.rs

```diff
@@ -377,6 +386,196 @@ impl From<IpV6Address> for [u16; IPV6_LEN / 2] {
     }
 }
 
+//= https://www.rfc-editor.org/rfc/rfc8200#section-3
+//#   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//#   |Version| Traffic Class |           Flow Label                  |
+//#   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//#   |         Payload Length        |  Next Header  |   Hop Limit   |
+//#   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//#   |                                                               |
+//#   +                                                               +
+//#   |                                                               |
+//#   +                         Source Address                        +
+//#   |                                                               |
+//#   +                                                               +
+//#   |                                                               |
+//#   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//#   |                                                               |
+//#   +                                                               +
+//#   |                                                               |
+//#   +                      Destination Address                      +
+//#   |                                                               |
+//#   +                                                               +
+//#   |                                                               |
+//#   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+define_inet_type!(
+    pub struct Header {
```

Comment:
It might just be me, but I read this right after the `ipv4` file and read `vtf` as `Version, TraFfic class`, and then was about to leave a comment about why we weren't including the flow label. I'm not sure what the clearer naming convention would be.

Reply:
I expanded it to `vtcfl` in my followup